rdr_service/gcloud_functions/retention_eligible_import_function/main.py

```diff
@@ -42,10 +42,14 @@ def created(self):
             "upload_date": self.event.timeCreated,
         }
 
-        _logger.info("Pushing cloud task...")
+        # check to see if the file being dropped in the bucket is the retention file before executing the function
+        if 'retention' in self.event.name:
+            _logger.info("Pushing cloud task...")
 
-        _task = GCPCloudTask()
-        _task.execute('/resource/task/ImportRetentionEligibleFileApi', payload=data, queue=task_queue)
+            _task = GCPCloudTask()
+            _task.execute('/resource/task/ImportRetentionEligibleFileApi', payload=data, queue=task_queue)
+        else:
+            _logger.info(f"skipping file: {self.event.name}, as it is not the retention file.")
```

Comment:
Why is this in the PR? We should follow a normal branch flow of branching off of `devel` and committing new changes on the branch.

Reply:
Yes, my mistake. this was an accident. I will branch of devel first in future.